### ISSUE TITLE
[1.10.2] Missing textures

### ISSUE THREAD

**Issue by serialtasted** [TechReborn/TechReborn#1445]
Hey there,

I have found some previous closed issues about this where you say it has been fixed (as referenced here: https://github.com/TechReborn/TechReborn/issues/817). But I'm using the latest version of TR (2.1.19.240) and Forge (12.18.3.2511) and I still have some missing textures.

Missing textures:
- Heat Generator
- Yellow Garnet Block
- Red Garnet Block
- Refined Iron Block

**Comment by modmuss50** [comment]
sorry, 1.10.2 is no longer supported expect for major issues.

**Comment by serialtasted** [comment]
Since you're not updating I made very simple fix for this problem. It doesn't fix the Heat Generator not having textures. I'll see if I can work that out with only the resource pack in the future, if I have the time.
At least now I have textures for Yellow Garnet, Red Garnet and Refined Iron blocks.

If anyone stumps across this issue, wanting a simple fix, there you go: (this is a resource pack)
[TR-Fix-1.10.2.zip](https://github.com/TechReborn/TechReborn/files/1777052/TR-Fix-1.10.2.zip)

For the Heat Generator I simple disabled it with modtweaker and hide it from jei:
```
// Remove Heat Generator and hide it
recipes.remove(<techreborn:heatgenerator>);
mods.jei.JEI.hide(<techreborn:heatgenerator>);
```
